### PR TITLE
Improve description for frequentis task2

### PR DESCRIPTION
**Pull request by pbochis**

### CODE REVIEW

File: controllers/mock.go [hunk 1]
```diff
@@ -341,10 +341,12 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
...
 		Assignment: model.Assignment{
 			Name:        "AvlTree",
-			Description: "AvlTree",
-			Instructions: "Your task is to write Junit unit tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation.\n" +
+			Description: "Testing an Avl Tree",
+			Instructions: "Your task is to write tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation. You should be familiar with writing java unit tests using the [JUnit Framework](http://junit.org/)\n" +
+				"We provide an AvlTree implementation and your responsibility is to test its logic. \n" +
+				"In order to successfully finish this task your unit tests should completely cover the specification of the AvlTree class. \n" +
 				"\n" +
-				"Below you are given the signatures and descriptions of all public operations of the AvlTree class.\n" +
```
Comment (flowlo) [inline controllers/mock.go]: Backticks please.

File: controllers/mock.go [hunk 2]
```diff
@@ -341,10 +341,12 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
 	taskTwo, err := model.Task{
 		Assignment: model.Assignment{
 			Name:        "AvlTree",
-			Description: "AvlTree",
-			Instructions: "Your task is to write Junit unit tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation.\n" +
+			Description: "Testing an Avl Tree",
+			Instructions: "Your task is to write tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation. You should be familiar with writing java unit tests using the [JUnit Framework](http://junit.org/)\n" +
```
Comment (flowlo) [inline controllers/mock.go]: "AVL tree", "AvlTree" is just the class name, not the name of the thing in general.

Comment (flowlo) [inline controllers/mock.go]: "Java", not "java".

File: controllers/mock.go [hunk 3]
```diff
@@ -341,10 +341,12 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
...
+				"We provide an AvlTree implementation and your responsibility is to test its logic. \n" +
+				"In order to successfully finish this task your unit tests should completely cover the specification of the AvlTree class. \n" +
```
Comment (flowlo) [inline controllers/mock.go]: Backticks please.

File: index.yaml [hunk 4]
```diff
@@ -8,7 +8,7 @@ indexes:
 # saying "# AUTOGENERATED").  If you want to manage some indexes
 # manually, move them above the marker line.  The index.yaml file is
 # automatically uploaded to the admin console when you next deploy
-# your application.
+# your application using appcfg.py.
```
Comment (flowlo) [inline index.yaml]: WTF

File: controllers/mock.go [hunk 5]
```diff
@@ -341,10 +341,12 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
...
 			Name:        "AvlTree",
-			Description: "AvlTree",
-			Instructions: "Your task is to write Junit unit tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation.\n" +
+			Description: "Testing an Avl Tree",
+			Instructions: "Your task is to write tests for an [AVL Tree](https://en.wikipedia.org/wiki/AVL_tree) implementation. You should be familiar with writing Java unit tests using the [JUnit Framework](http://junit.org/)\n" +
+				"We provide an AVL Tree implementation and your responsibility is to test its logic. \n" +
+				"Your test suite will be run against multiple versions of the `AvlTree` class, some of which are bugged. Make sure that you cover all possible cases\n" +
 				"\n" +
-				"Below you are given the signatures and descriptions of all public operations of the AvlTree class.\n" +
+				"Below you are given the signatures and descriptions of all public operations of `AvlTree.java`:\n" +
```
Comment (flowlo) [inline controllers/mock.go]: Why ".java"?

**Comment by Exordian** [comment]
rewritten in 9c8269df8009753153ce383cd3615e0064025016

**Pull request closed without merging**